gmso/tests/files/restrained_benzene_ua.top

```diff
@@ -0,0 +1,76 @@
+; File Topology written by GMSO at 2022-10-25 13:20:26.887847
+
+[ defaults ]
+; nbfunc	comb-rule	gen-pairs		fudgeLJ	fudgeQQ
+1			2			no		0.0		0.0
+
+[ atomtypes ]
+; name		at.num	mass		charge		ptype	sigma	epsilon
+CH_sp2      6       13.01900     0.00000	A        0.36950     0.41988
+
+[ moleculetype ]
+; name	nrexcl
+Compound	3
+[ atoms ]
+; nr	type	resnr	residue		atom	cgnr	charge	mass
+1       CH_sp2      1       Compound    C       1        0.00000    13.01900
+2       CH_sp2      1       Compound    C       1        0.00000    13.01900
+3       CH_sp2      1       Compound    C       1        0.00000    13.01900
+4       CH_sp2      1       Compound    C       1        0.00000    13.01900
+5       CH_sp2      1       Compound    C       1        0.00000    13.01900
+6       CH_sp2      1       Compound    C       1        0.00000    13.01900
+
+[ bonds ]
+; ai	aj	funct	c0	c1
+2       1       1           0.14000        0.00000
+6       1       1           0.14000        0.00000
+3       2       1           0.14000        0.00000
+4       3       1           0.14000        0.00000
+5       4       1           0.14000        0.00000
+6       5       1           0.14000        0.00000
+
+[ angles ]
+; ai	aj	ak	funct	c0	c1
+2       1       6       1         120.00000        0.10000
+1       6       5       1         120.00000        0.10000
+4       5       6       1         120.00000        0.10000
+1       2       3       1         120.00000        0.10000
+3       4       5       1         120.00000        0.10000
+2       3       4       1         120.00000        0.10000
+
+[ angle_restraints ]
+; ai	aj	ai	ak	funct	theta_eq	k	multiplicity
+2       1       6       1         120.00000        0.10000
```

Comment:
Should theta_eq be 1000, not 120?

Reply:
theta is angle so 120 looks ok to me

Reply:
Sorry I meant, 292.88000 for the k-value.  as it is input as 1000, in the .py file:

```
for angle in top.angles:
    print(f"angle = {angle}")
    # Apply restraint for angle
    angle.restraint = {
        "theta_eq": angle.angle_type.parameters["theta_eq"],
        "k": 1000 * u.kJ / u.mol,
        "n": 1,
```
    }

Reply:
I guess I dont really understand what the "k" and "n" values above are doing?

Reply:
This issue should be fixed now, the k supposed to be the force that maintain the theta_eq angle (so restraining the angle, not fixed it), the n is multiplicity of that angle.

Reply:
OK.  

My comments are referring to this manual page https://manual.gromacs.org/current/reference-manual/topologies/topology-file-formats.html?highlight=angle_restraints

I guess this is a little confusing because above in the GROMACS manual, they have a "restricted bending potential," which does not have a multiplicity.  Not sure what the multiplicity actually does either... 

In the GROMACS manual, the angle_straints one says with 4 atoms also ??   I assume this is a typo..